nautobot/ipam/models.py

```diff
@@ -976,7 +977,7 @@ class VLANGroup(OrganizationalModel):
     """
 
     name = models.CharField(max_length=100)
-    slug = models.SlugField(max_length=100)
+    slug = AutoSlugField(populate_from="name", max_length=100, editable=True, overwrite_on_add=False)
```

Comment:
:sad: about non-globally-unique slug. I may open an issue about these.

Reply:
Yeah, this one I don't think uses unique_together, unlike rack and devicetype.